src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -4663,6 +4663,7 @@ public function has_class( $wanted_class ): ?bool {
 	 *     // Outputs: "free <egg> lang-en "
 	 *
 	 * @since 6.6.0 Subclassed for the HTML Processor.
+	 * @since 6.7.0 Null bytes are replaced with the replacement character (U+FFFD).
```

Comment:
I'd consider this just bug-fixing the method instead of being specifically about null bytes. We still aren't replacing invalid UTF-8 sequences, which in my mind fits into the same category. Given that we plan on auditing all of the CSS semantics, I would imagine we could leave a note more in line with that.

```
 * @since 6.7.0 Fixes for improved CSS support.
```

but then again, I don't know if I have generally thought that `@since` tags for bug-fixes add much value unless there's a notable change. every time we encounter null bytes we are dealing with invalid HTML, while proper class names don't change.

Reply:
I'm happy to remove it.

```suggestion
```